views/RandomBeer.jsx

```diff
@@ -0,0 +1,35 @@
+const React = require('react')
+const Layout = require('./Layout')
+
+function RandomBeer(props) {
+    console.log(props)
+    return (
+      <Layout>
+        <main>
+          <h1>Random Beer</h1>
+          <div>
+            {props.randomBeer.map((beerObj, i) =>{
+                return(
+                  <div key = {i}>
+                    <img src={beerObj.image_url} width="200" alt="" />
+                    <h3>{beerObj.name}</h3>
+                    <h5>{beerObj.tagline}</h5>
+                    <p>{beerObj.description}</p>
+                    <p>{beerObj.food_pairing}</p>
```

Comment:
For the food pairing, you can display this better using a map. Like this:

`<ul>{beerObj.food_pairing.map((oneFood) => {
                      return <li>{oneFood}</li>
                    })}</ul>`